Lib/test/test_dbm_gnu.py

```diff
@@ -32,7 +32,7 @@ def test_disallow_instantiation(self):
         # Ensure that the type disallows instantiation (bpo-43916)
         self.g = gdbm.open(filename, 'c')
         tp = type(self.g)
-        self.assertRaises(TypeError, tp)
+        support.check_disallow_instantiation(self, tp)
```

Comment:
I suggest to write directly: `support.check_disallow_instantiation(self, type(self.g))`